packages/lwc/scripts/build.js

```diff
@@ -13,23 +13,11 @@ const distDirectory = path.join(__dirname, '../dist');
 
 const COMMON_TARGETS = [
     // ESM
-    { target: 'es2017', format: 'esm', prod: false },
+    { target: 'es2017', format: 'esm' },
 
     // IIFE
-    { target: 'es5', format: 'iife', prod: false },
-    { target: 'es5', format: 'iife', prod: true },
-    { target: 'es5', format: 'iife', prod: true, debug: true },
-    { target: 'es2017', format: 'iife', prod: false },
-    { target: 'es2017', format: 'iife', prod: true },
-    { target: 'es2017', format: 'iife', prod: true, debug: true },
-
-    // UMD
-    { target: 'es5', format: 'umd', prod: false },
-    { target: 'es5', format: 'umd', prod: true },
-    { target: 'es5', format: 'umd', prod: true, debug: true },
-    { target: 'es2017', format: 'umd', prod: false },
-    { target: 'es2017', format: 'umd', prod: true },
-    { target: 'es2017', format: 'umd', prod: true, debug: true },
+    { target: 'es5', format: 'iife' },
+    { target: 'es2017', format: 'iife' },
```

Comment:
We could also remove the `es5` targets, and just use ES5 internally (e.g. in our karma tests). Maybe it's time to do this as well, since even `lwc-platform` is not using these files.